pkg/test/env/istio.go

```diff
@@ -59,6 +59,9 @@ var (
 	// nolint: golint
 	PULL_POLICY Variable = "PULL_POLICY"
 
+	// VERSION is Docker tag used for the base_debug image.
+	VERSION Variable = "1.3.0-dev"
```

Comment:
would be nice to not have this set in multiple places - we had issues with having multiple places go out of sync before

Reply:
just for testing in the gate at present.  I will clean this particular point up, but I totally get what your saying.

The makefile should pull this VERSION variable from the environment - which makes me think for testing I need to set this to something invalid.

Reply:
I think in the makefile we have `VERSION ?= 1.3-dev` so it can be overriden but has a default

Reply:
That is what I had meant - but that varisble should be used here, rather than my current hardcode.  I am simply attempting to get what I have to build properly and past the various checks :)